selfdrive/car/honda/carstate.py

```diff
@@ -155,19 +155,13 @@ def get_can_signals(CP, gearbox_msg="GEARBOX"):
     ]
   elif CP.carFingerprint in (CAR.CRV, CAR.CRV_EU, CAR.ACURA_RDX, CAR.PILOT_2019, CAR.RIDGELINE):
     signals += [("MAIN_ON", "SCM_BUTTONS", 0)]
-  elif CP.carFingerprint == CAR.FIT:
-    signals += [("CAR_GAS", "GAS_PEDAL_2", 0),
-                ("MAIN_ON", "SCM_BUTTONS", 0),
-                ("BRAKE_HOLD_ACTIVE", "VSA_STATUS", 0)]
-    checks += [
-      ("GAS_PEDAL_2", 100),
-    ]
-  elif CP.carFingerprint == CAR.HRV:
-    signals += [("CAR_GAS", "GAS_PEDAL", 0),
+  elif CP.carFingerprint in (CAR.FIT, CAR.FREED, CAR.HRV):
+    gas_pedal = "GAS_PEDAL" if CP.carFingerprint == CAR.HRV else "GAS_PEDAL_2"
+    signals += [("CAR_GAS", gas_pedal, 0),
```

Comment:
By the way, on HRV, this `CAR_GAS` signal is unused by carstate, which seems like a bug.  I think it should be used in the following code (rather than grouping HRV with CRV):

```python
HRV has vl["GAS_PEDAL"]["CAR_GAS"] available, so seems like it should be used.

self.pedal_gas = cp.vl["POWERTRAIN_DATA"]["PEDAL_GAS"]
# crv doesn't include cruise control
if self.CP.carFingerprint in (CAR.CRV, CAR.CRV_EU, CAR.HRV, CAR.ODYSSEY, CAR.ACURA_RDX, CAR.RIDGELINE, CAR.PILOT_2019, CAR.ODYSSEY_CHN):
  ret.gas = self.pedal_gas / 256.
else:
  ret.gas = cp.vl["GAS_PEDAL_2"]["CAR_GAS"] / 256.
```